expressEngine/product/service/product.js

```diff
@@ -7,13 +7,16 @@ import { Product, ProductDetail } from '../../model';
 // import messages
 import {success} from '../../cms/product';
 
+//import validateRequired
+import { validateRequired } from '../../lib/validationHandler';
+
 class Service extends BaseService {
   registerProduct = async (data) => {
     try {
       const {name, supplierId,unitPrice, productPackage, isDiscontinued  } = data,
```

Comment:
const {name, supplierId,unitPrice, productPackage, isDiscontinued  } = data,
fix eslint error.
is should be
const { name, supplierId,unitPrice, productPackage, isDiscontinued } = data,

Reply:
you also forgot to add space between supplierId and unitPrice